task/host-sp-comms/src/main.rs

```diff
@@ -1558,6 +1575,10 @@ cfg_if::cfg_if! {
         // gimletlet doesn't have an SP3 to interrupt, but we can wire up an LED
         // to one of the exposed E2-E6 pins to see it visually.
         const SP_TO_SP3_INT_L: sys_api::PinSet = sys_api::Port::E.pin(2);
+    } else if #[cfg(target_board = "grapefruit")] {
+        // we don't have an SP3 interrupt pin, so pick an unconnected GPIO
+        // TODO this is specific to SP3; but SP5 is coming!
```

Comment:
Both!  We should probably rename the constant to something like `SP_TO_HOST_CPU_INT_L`, but we also don't have any SP5 boards that actually use this pin (since Grapefruit doesn't have it wired up).  That will come in a future `#[cfg(target_board = "cosmo")]` block.

Reply:
On second though, I can do the renaming here (552e1955ee6b)